resfit/fit_resonator/fit_functions.py

```diff
@@ -27,36 +26,38 @@ def from_params(cls, Qi,  Qc, f_res, phi, Q=None):
         Qa = Qi/f_res
         return cls(Qi=Qi, Qc=Qc, f_res=f_res,phi=phi, Q=Q, kappa=kappa, Qa=Qa)
 
-
-class FittingMethod(enum.Enum):
-    DCM = 1
-    DCM_REFLECTION = 2
-    PHI = 3
-    INV = 4
-    CPZM = 5
-
-def Cavity_DCM(x, Q, Qc, w1, phi):
+def cavity_DCM(x, Q, Qc, w1, phi):
     #DCM fit function
     return np.array(1-Q/Qc*np.exp(1j*phi)/(1 + 1j*(x-w1)/w1*2*Q))
 
-def Cavity_DCM_REFLECTION(x, Q, Qc, w1, phi):
+def cavity_DCM_REFLECTION(x, Q, Qc, w1, phi):
     #DCM REFLECTION fit function
     return np.array(1-2*Q/Qc*np.exp(1j*phi)/(1 + 1j*(x-w1)/w1*2*Q))
 
-def Cavity_inverse(x, Qi, Qc, w1, phi):
+def cavity_inverse(x, Qi, Qc, w1, phi):
     #Inverse fit function
     return np.array(\
         (1 + Qi/Qc*np.exp(1j*phi)/(1 + 1j*2*Qi*(x-w1)/w1)))
 
-def Cavity_CPZM(x, Qi, Qic, w1, Qia):
+def cavity_CPZM(x, Qi, Qic, w1, Qia):
     #CPZM fit function
     return np.array(\
         (1 + 2*1j*Qi*(x-w1)/w1)/(1 + Qic +1j*Qia + 1j*2*Qi*(x-w1)/w1))
 
-def One_Cavity_peak_abs(x, Q, Qc, w1):
+def one_cavity_peak_abs(x, Q, Qc, w1):
     #Ideal resonator fit function
     return np.abs(Q/Qc/(1 + 1j*(x-w1)/w1*2*Q))
 
-def One_Cavity_peak_abs_REFLECTION(x, Q, Qc, w1):
+def one_cavity_peak_abs_REFLECTION(x, Q, Qc, w1):
     #Ideal resonator fit function
-    return np.abs(2*Q/Qc/(1 + 1j*(x-w1)/w1*2*Q))
\ No newline at end of file
+    return np.abs(2*Q/Qc/(1 + 1j*(x-w1)/w1*2*Q))
```

Comment:
Doesn't look like either of the `one_cavity_` methods are used -- maybe add them to the enum?